File: Programming Basics with JS/T04 - Nested Conditional Statements/exercise/09. Ski Trip.js
function ski(input){
    let days = Number(input[0]);
    let typeOfHousing = (input[1]);
    let feedback = input[2];
    let nights = Number(days - 1);
    let price = 0;

    if (days < 10) {
        switch(typeOfHousing) {
            case "room for one person" :
                price = 18.00 * nights;
                break;
            case "apartment" :
                price = 25 * 0.70 * nights;
                break;
            case "president apartment" :
                price = 35 * 0.90 * nights;
                break;

        } if (feedback === "positive") {
            price = price * 1.25;
        } else if (feedback === "negative") {
            price = price * 0.90;
        }

    } else if (days >= 10 && days <= 15) {
        switch(typeOfHousing){
            case "room for one person" :
                price = 18.00 * nights;
                break;
            case "apartment" :
                price = 25 * 0.65 * nights;
                break;
            case "president apartment" :
                price = 35 * 0.85 * nights;
                break;

        } if (feedback === "positive") {
            price = price * 1.25;
        } else if (feedback === "negative") {
            price = price * 0.90;
        }

    } else {
        switch(typeOfHousing) {
            case "room for one person" :
                price = 18.00 * nights;
                break;
            case "apartment" :
                price = 25 * 0.50 * nights;
                break;
            case "president apartment" :
                price = 35 * 0.80 * nights;
                break;

        } if (feedback === "positive") {
            price = price * 1.25
        } else if (feedback === "negative") {
            price = price * 0.90;
        }
    }
    console.log(price.toFixed(2));
}
ski(["14", "apartment", "positive"]);

File: Programming Basics with Python/T04 - Conditional Statements (Advanced)/exercise/09. Ski Trip.py
days = int(input())
room_type = input()
grade = input()

room_price = 18.00
apartment_price = 25.00
presidential_price = 35.00
nights = days - 1
total_price = 0

if days < 10:
    if room_type == "room for one person":
        total_price = room_price * nights
    elif room_type == "apartment":
        total_price = apartment_price * nights * 0.70
    elif room_type == "president apartment":
        total_price = presidential_price * nights * 0.90

elif 10 <= days <= 15:
    if room_type == "room for one person":
        total_price = room_price * nights
    elif room_type == "apartment":
        total_price = apartment_price * nights * 0.65
    elif room_type == "president apartment":
        total_price = presidential_price * nights * 0.85

elif days > 15:
    if room_type == "room for one person":
        total_price = room_price * nights
    elif room_type == "apartment":
        total_price = apartment_price * nights * 0.50
    elif room_type == "president apartment":
        total_price = presidential_price * nights * 0.80

if grade == "positive":
    total_price = total_price * 1.25

elif grade == "negative":
    total_price = total_price * 0.90

print(f"{total_price:.2f}")